test/events/test_maintain_orcid_webhook.py

```diff
@@ -90,5 +96,50 @@ def test_it_ignores_other_models_being_committed(orcid_token: OrcidToken,
     assert mock_orcid_client.remove_webhook.call_count == 0
 
 
-def test_it_has_a_valid_signal_handler_registered_on_app(registered_handler_names: List[str]):
-    assert 'webhook_maintainer' in registered_handler_names
+@patch('profiles.events.catch_exceptions')
+def test_exception_not_handled_if_catch_decorator_is_removed(mock_catch: MagicMock,
+                                                             profile: Profile,
+                                                             orcid_config: Dict[str, str],
+                                                             mock_orcid_client: MagicMock,
+                                                             session: scoped_session,
+                                                             url_safe_serializer:
+                                                             URLSafeSerializer):
+    with pytest.raises(Exception):
```

Comment:
This shouldn't cover the whole test? Removing it sees an earlier exception raised.